microservices/nris_api/backend/requirements.txt

```diff
@@ -0,0 +1,45 @@
+alembic==1.0.0
```

Comment:
I know these started off as a copy of the python backend requirements. I did an audit on those and more than half requirements we were able to get rid of and upgrade the rest.
Can you remove the ones you're not using from here?

Reply:
How about this, i'm gonna comment out a bunch when we start working on this a bunch, and then we will uncomment if they are needed or delete them if it never comes up. I don't really want to spend overhead time just testing this (there are no tests or things that use these libraries, ever though they will in the next two weeks.